src/test/ui/lint/rfc-2383-lint-reason/catch_multiple_lint_triggers.rs

```diff
@@ -0,0 +1,45 @@
+// check-pass
```

Comment:
Note for me to review the UI tests more thoroughly at the end of the commit set.